serverless.yml.example

```diff
@@ -0,0 +1,72 @@
+
+# MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+service: dca-bot-PUT_MARKET_NAME_HERE
+#< END MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: python3.9
+  memorySize: 128 # optional, in MB, default is 1024
+  timeout: 20 # optional, in seconds, default is 6
+  lambdaHashingVersion: 20201221
+  stage: ${opt:stage}
+  region: us-east-1
+
+package:
+  exclude:
+    - .gitignore
+    - .github
+    - __tests__/**
+    - fixtures/**
+    - setupJest.js
+    - 'README.md'
+    - '*.iml'
+    - '*.example'
+
+functions:
+  buy:
+    handler: handler.buy
+    events:
+      # MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+      # Run at 0:00 UTC every day
+      - schedule: cron(0 0 * * ? *)
+      #< END MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    environment:
+      ENV: ${opt:stage}
+      # MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+      MARKET_NAME: 'BTC-USD'  # (e.g. BTC-USD, ETH-BTC, etc)
+      AMOUNT: 100             # The quantity to buy or sell in the amount_currency
+      AMOUNT_CURRENCY: 'USD'  # The currency the amount is denominated in
+      #< END MAKE EDITS HERE !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
How about

```
- schedule: cron(0 0 * * ? *)
- input: '[{"marketName":"BTC-USD","amount":100,"amountCurrency":"USD"},{"marketName":"ETH-USD","amount":100,"amountCurrency":"USD"}]'
```
We could then only have one yaml file for as many trading pairs as needed?